app/Models/Setting.php

```
@@ -22,6 +23,16 @@ class Setting extends Model
         'end_time',
     ];
 
+    public static function cached(): ?self
+    {
+        return Cache::remember('app_settings', 3600, fn () => static::first());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'cached'. (undefined)

(StaticAccess)